Makefile

```diff
@@ -10,6 +10,9 @@ PACKER_WINDOWS_FILES = $(exec find packer/windows)
 AWS_REGION ?= us-east-1
 AMZN_LINUX2_AMI ?= $(shell aws ec2 describe-images --region $(AWS_REGION) --owners amazon --filters 'Name=name,Values=amzn2-ami-hvm-2.0.????????-x86_64-gp2' 'Name=state,Values=available' --output json | jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId')
 
+ARM64_INSTANCE_TYPE = m6g.xlarge
+X86_INSTANCE_TYPE = c5.xlarge
```

Comment:
Hmm. We use AMD64 rather than X86_64 everywhere else.
However this file deals with upstream Amazon Linux 2 AMIs, which use x86_64.
I wonder if this variable should be:
* `X86_INSTANCE_TYPE` (current, introduced in this PR)
* `X86_64_INSTANCE_TYPE` (Because X86 != X86_64, and aligned with AWS / Amazon Linux 2)
*`AMD64_INSTANCE_TYPE` (aligned with all the references that we control)

Very minor detail, it probably doesn't matter.